roles/logs_server/templates/logstoprm_staging.sh.j2

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+# {% raw %}
+
+set -eE
+# Everything is logged in the system logs with logger
+trap 'logger -t logstoprm-cron "The command (${BASH_COMMAND}) had failed on fine number (${LINENO})"' ERR
+
+{% endraw %}
+
+# Where the logs will be stored to
+destination_dir="{{ logstoprm_rsyncd_module.rsyncd_dir }}"
+
+# {% raw %}
+
+# Where the logs will be collected
+source_dir="/var/log/remote/"
+
+# check the rest of environment
+test -d "${source_dir}" || \
+  { echo "error directory ${source_dir} does not exist or has wrong permissions" ; exit 1; }
+test -d "${destination_dir}" || \
+  { echo "error directory ${destination_dir} does not exist or has wrong permissions" ; exit 1; }
+cd "${source_dir}"
+
+# Description
+# - find compressed files older than 7 days
+# - ignore files smaller than 20-bytes as they are compressed empty files (plus
+#   20 bytes can not contain enough information to help with anything)
+# - move them (that is: copy and then delete)
+# - and change ownership: owner=root and group=logstoprm
+# - and change permissions for user and group to be able to read-only
+# find flags
+#   --remove-source-files  after they have been successfully copied
+#   -R use                 relative path-names
+#   +21c                   more than 21 characters (bytes)
+# rsync flags
+#   --ignore-existing      don't update files if they already exist on destination
+logger -t logstoprm "Starting to move log files to ${destination_dir}"
+find ./ -type f -size +21c -mtime +7 -name "*.gz" -exec rsync \
+   --remove-source-files --ignore-existing -a -R --min-size=21 \
+   --chown=root:{% endraw %}{{ logstoprm_rsyncd_module.group }}{% raw %} \
```

Comment:
To prevent multiple `{% endraw %} some exception {% raw %}` blocks in the Bash code I suggest to use a mapping from Jinja to Bash variables at the top of the script followed by a comment stating there shall be no more Jinja vars after that comment and that everything after the comment is wrapped in a single large raw block. E.g.
```
#!/bin/bash

declare some_bash_var='{{ some_jinja_var }}'
declare another_bash_var='{{ another_jinja_var }}'
#
# All Jinja vars must be mapped to bash vars above;
# There shall be no more Jinja below this comment.
#
{% raw %}
# Lots of bash code without Jinja complications.
echo "${some_bash_var}"
echo "${another_bash_var}"
{% endraw %}
```